tools/github-actions/new-version/packaged-action/index.js

```diff
@@ -34134,23 +34133,22 @@ class NewVersion {
      * @param versionMask
      */
     computeNewVersion(tags, versionMask) {
-        const regexpVersionMask = new RegExp(`^(?:v)?${versionMask}`);
         // Sort tags in descending order and exclude next major in preparation
         let parsedSortedTags = tags
             .map((tag) => semver.parse(tag.replace('V', 'v')))
-            .filter((tag) => !!tag && !!regexpVersionMask.test(tag.raw))
+            .filter((tag) => !!tag && !!tag.raw.match(`^(?:v)?${versionMask}`))
```

Comment:
The previous version was more efficient, why this change?

Reply:
issue coming from the build of tools 
it will be fixed after the rebase 
I fallback all changes from tools